### PR TITLE
set general styles

### PR DESCRIPTION
**Pull request by joer9514**
fix frontend styles

### CODE REVIEW

File: packages/core/src/base/_reset.scss
Comment (adonyssantos) [inline packages/core/src/base/_reset.scss:1]: For some reason, the file you modified in PR #39 is also reflected here. You may have created this branch from `style/reset-css` and not from `release/2.0.0.0`. Keep this in mind for next time.

File: packages/core/src/base/_generics.scss [hunk 1]
```diff
@@ -1,15 +1,6 @@
...
-p {
-    line-height: 27px;
-}
-
-img {
-    vertical-align: top;
-    width: 100%;
+  color: $color_dark;
+  background-color: $color_light;
+  font-family: $font;
```
Comment (adonyssantos) [inline packages/core/src/base/_generics.scss:4]: This variable name is so generic. I think that we will create a more descriptive variable name, like `$font-body`, `$font-headings`, etc.

**Review by adonyssantos — APPROVED** [review]
Good to merge. 🚀
We need to work on some changes in another branch.

File: packages/core/src/base/_reset.scss
Comment (joer9514) [reply packages/core/src/base/_reset.scss:1]: Ok

**Pull request merged**